src/develop/blends/blendif_lab.c

```diff
@@ -1362,93 +1362,128 @@ static _blend_row_func *_choose_blend_func(const unsigned int blend_mode)
 }
 
 
+#ifdef _OPENMP
+#pragma omp declare simd aligned(sRGB, XYZ: 16)
+#endif
+static inline void _linear_sRGB_to_XYZ(const float *const restrict sRGB, float *const restrict XYZ)
```

Comment:
linear sRGB is Rec709, there is a function for that in `colorspaces_inline_conversion.h`, unless the white point does not match.

Reply:
Thanks :) Yes the white point matches the ones of the function `dt_XYZ_to_Rec709_D50`, used that one instead.